docs/index.rst

```diff
@@ -45,9 +45,9 @@ Install Via Git
 * Install `Docker <https://www.docker.com>`__
 * Install `Cookiecutter <https://cookiecutter.readthedocs.io/en/latest/installation.html#install-cookiecutter>`__
 * Clone this repo: ``git clone https://github.com/bhdouglass/clickable.git``
-* Add clickable to your PATH: ``echo "export PATH=\$PATH:\$HOME/clickable" >> ~/.bashrc``
+* Add clickable to your PATH: ``echo "export PATH=$PATH:$HOME/clickable" >> ~/.bashrc``
```

Comment:
Actually I think we want the slashes, that way the shell doesn't expand them (since this is going in the user's .bashrc and they might have already made changes to PATH)

Reply:
Oops! You're right. Sorry, I didn't realize that it was intended to be copied in the .bashrc